crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -972,6 +973,29 @@ fn handle_dict_unpacking_comment<'a>(
     }
 }
 
+/// Handle comments between a function call and its arguments. For example, attach the following as
+/// dangling on the call:
+/// ```python
+/// (
+///   func
+///   # dangling
+///   ()
+/// )
+/// ```
+fn handle_call_comment(comment: DecoratedComment) -> CommentPlacement {
+    if comment.line_position().is_own_line() {
+        if comment.preceding_node().is_some_and(|preceding| {
+            comment.following_node().is_some_and(|following| {
+                preceding.end() < comment.start() && comment.end() < following.start()
+            })
+        }) {
```

Comment:
Nit:

I find this nesting hard to read. 
```suggestion
		if let (Some(preceding), Some(following)) = (comment.preceding_node(), comment.following_node()) {
			if TextRange::new(preceding.end(), following.start()).contains(comment.range()) {
```